Pinta.Core/Extensions/DialogExtensions.cs

```diff
@@ -0,0 +1,84 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Gtk;
+
+namespace Pinta.Core
+{
+	public static class DialogExtensions
+	{
+		public static ResponseType ShowErrorDialog (string markup)
+			=> ShowMessageDialog (markup, MessageType.Error, ButtonsType.Ok);
+
+		public static ResponseType ShowMessageDialog (string markup)
+			=> ShowMessageDialog (markup, MessageType.Info, ButtonsType.Ok);
+
+		public static ResponseType ShowQuestionDialog (string markup, params DialogButton[] buttons)
+		{
+			using (var md = CreateMessageDialog (markup, MessageType.Question, ButtonsType.None)) {
+				foreach (var button in SortButtonsForOS (buttons)) {
+					md.AddButton (button.Text, button.Response);
+
+					if (button.IsDefault)
+						md.DefaultResponse = button.Response;
+				}
+
+				return (ResponseType) md.Run ();
+			}
+		}
+
+		private static ResponseType ShowMessageDialog (string markup, MessageType type, ButtonsType buttons, ResponseType? defaultResponse = null)
+		{
+			using (var md = new MessageDialog (PintaCore.Chrome.MainWindow, DialogFlags.Modal, type, buttons, true, markup)) {
```

Comment:
Could this call into `CreateMessageDialog()` to avoid a little bit of duplication?

Reply:
Doh, I meant to change that. Fixed!